catalog-spec/catalog-recommendations.md

```diff
@@ -4,8 +4,89 @@ This document represents a number of ideas for options and requirements that are
 Some will likely evolve in to requirements, or at least documented specification extensions. But there
 is not yet enough consensus on how to represent them.
 
+### Metadata Linking
+
+In general STAC aims to be oriented around **search**, centered on the core fields that users will want to search on to find imagery.
+The core is space and time, but there are often other metadata attributes that are useful. While the specification is flexible enough that
+providers can fill it with tens or even hundreds of fields of metadata that is not recommended. If providers have lots of metadata then 
+that should be linked to in an 'asset', indeed even a new json asset that is potentially easier to parse. There is a lot of metadata that
+is only of relevance to advanced processing algorithms, and while that is important it should not be in the core STAC items.
+
+
+### Prefixes
+
+A spatiotemporal asset catalog item can combine schema information from several different sources - the core STAC item information, 
+an earth observation community extension, and a vendor specific provider. It can be difficult to distinguish exactly where each definition
+came from, and to pull out the most relevant information, especially when vendors often will dump in all the metadata they have in to the
+STAC definition. 
+
+So one idea is to have prefixes to differentiate specific vendors (like dg: for DigitalGlobe), and for communities of practice (like eo: for Earth 
+Observation). These wouldn't be full namespacing, though an extension for like JSON-LD could potentially evolve to make fully resolved namespacing
+an option. 
+
+An example of this can be seen in a Landsat example:
+
+```json
+
+  "properties": {
+    "start_datetime": "2014-06-02T2014-06-02",
+    "end_datetime": "2014-06-02T2014-06-02",
+    "provider": "LANDSAT_8",
+    "license": "PDDL-1.0",
+
+    "eo:off_nadir_angle" : -0.001,
+    "eo:cloud_cover" : 10.31,
+    "eo:sun_azimuth" : 149.01607154,
+    "eo:sun_elevation" : 59.21424700,
+    "eo:resolution" : 30,
+
+    "l8:data_type": "L1T",
+    "l8:wrs_path": 153,
+    "l8:wrs_row": 25,
+    "l8:earth_sun_distance": 1.0141560,
+    "l8:ground_control_points_verify": 114,
+    "l8:geometric_rmse_model": 7.562,
+    "l8:image_quality_tirs": 9,
+    "l8:ground_control_points_model": 313,
+    "l8:geometric_rmse_model_x": 5.96,
+    "l8:geometric_rmse_model_y": 4.654,
+    "l8:geometric_rmse_verify": 5.364,
+    "l8:image_quality_oli": 9
+
+  },
+
+```
+
+This could also extend to product level metadata:
```

Comment:
Why is it 'product' level? Overall, the following example seems confusing and outdated... bands vs eo:bands, other eo fields etc.

Reply:
I didn't get a chance to look in depth at everything that was in this document. It's in the repo now, so was just trying to make it a bit better. If there's not time to really go over it then we can just remove it. Best thing to do might be to move this particular discussion to an issue, since there's not really a great 'recommendation' there. I just wanted to make sure this bit wasn't lost, as it captured some of the previous ideas / options.